### PR TITLE
Hotfix/1.0.11

### PR DESCRIPTION
**Pull request by motin**
* Drop pings that are attempted to be sent while a private browsing window is open
* Removed debug log statements related to install/uninstall add-ons
* Bump version to 1.0.11

Fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1443172

### CODE REVIEW

File: addon/lib/Feature.jsm [hunk 1]
```diff
@@ -370,8 +371,27 @@ class Feature {
     this.telemetry(stringStringMap);
   }
 
-  /* good practice to have the literal 'sending' be wrapped up */
+  aPrivateBrowserWindowIsOpen() {
+    if (PrivateBrowsingUtils.permanentPrivateBrowsing) {
+      return true;
+    }
+    let win = null;
```
Comment (rhelmer) [inline addon/lib/Feature.jsm]: Hm, why is this needed? Looks like it isn't used.

File: addon/lib/Feature.jsm [hunk 2]
```diff
@@ -370,8 +371,27 @@ class Feature {
...
+    let windowList = Services.wm.getEnumerator("navigator:browser");
+    while (windowList.hasMoreElements()) {
+      let nextWin = windowList.getNext();
+      if (PrivateBrowsingUtils.isWindowPrivate(nextWin)) {
+        return true;
+      }
+      win = nextWin;
```
Comment (rhelmer) [inline addon/lib/Feature.jsm]: Seems like this isn't used.

File: addon/lib/Feature.jsm [hunk 3]
```diff
@@ -147,7 +148,7 @@ class Client {
         feature.notifyViaTelemetry(dataOut);
 
       } else if (addonChanges.lastDisabledOrUninstalled) {
-        feature.log.debug("Just disabled", client.lastDisabledOrUninstalled, "from", uri);
+        //feature.log.debug("Just disabled", client.lastDisabledOrUninstalled, "from", uri);
```
Comment (rhelmer) [inline addon/lib/Feature.jsm]: You might consider having some way at runtime to enable debugging logging (like watching a pref), or just a constant set so you have a single place to enable/disable this sort of thing :) Not a blocker though.

**Review by rhelmer — APPROVED** [review]
lgtm overall, I don't think the `win` variable is used and just a general comment on the logging thing, but I don't see any actual problems with this as-is.

File: addon/lib/Feature.jsm (hunk 3 above)
Comment (motin) [reply addon/lib/Feature.jsm]: Debug logging is already only enabled by pref, commenting this here is just an insane safety measure :)
